arches_her/views/map.py

```diff
@@ -0,0 +1,33 @@
+from django.views.generic import View
+from django.db import connection
+from django.http import HttpResponse, Http404
+from arches.app.models import models
+from arches.app.utils.permission_backend import get_restricted_instances
+from arches.app.search.search_engine_factory import SearchEngineFactory
+
+
+class ApplicationAreas(View):
+    def get(self, request, zoom, x, y):
+        se = SearchEngineFactory().create()
+        restricted_resource_ids = get_restricted_instances(request.user, search_engine=se)
+        if len(restricted_resource_ids) == 0:
+            restricted_resource_ids.append("10000000-0000-0000-0000-000000000001")  # This must have a uuid that will never be a resource id.
+        restricted_resource_ids = tuple(restricted_resource_ids)
+        nodeid = '1909956f-3a3b-11eb-ae99-f875a44e0e11'    
```

Comment:
@csmith-he where did the node permission check go? This only checks resource instance instance permissions